GameData/RealSolarSystem/Localization/fr-fr.cfg

```diff
@@ -0,0 +1,95 @@
+Localization
+{
+    fr-fr
+    {
+        //Planet & Biomes
+
+        // The genders of celestial bodies that do not have articles (that is,
+        // all but « le Soleil », « la Terre », and « la Lune ») are somewhat
+        // unclear.  From the gender of their namesakes, it appears to drift
```

Comment:
As discussed offline, Littré (1863-1873) uses systematically the gender of the eponymous divinity.  I would stick to that usage, as it's as good a linguistic reference as any other.  Arguably the usage is fluctuating, but we have to choose a normative reference.  FWIW, personally I find `Cérès^M` and `Jupiter^F` strange.

Reply:
This makes most easier, but then we have the moons of Uranus.